helm/o11y-canary/templates/configmap.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: o11y-canary-config
+  labels:
+    app: o11y-canary
+data:
+  config.yaml: |-
+    {{ .Values.canaryConfig | nindent 4 }}
```

```suggestion
    {{- .Values.canaryConfig | nindent 4 }}
```
we should strip leading whitespace, this will avoid an extra new line before the actual config:

Before:
```
data:
  config.yaml: |-

    canary:
      my_canary_1:
```

After:
```
data:
  config.yaml: |-
    canary:
      my_canary_1:
```